packages/cli/src/commands/doctor/healthchecks/__tests__/androidSDK.test.ts

```diff
@@ -0,0 +1,78 @@
+import androidSDK from '../androidSDK';
+import getEnvironmentInfo from '../../../../tools/envinfo';
+import {EnvironmentInfo} from '../../types';
+import {NoopLoader} from '../../../../tools/loader';
+
+import * as common from '../common';
+
+const logSpy = jest.spyOn(common, 'logManualInstallation');
+
+describe('androidHomeEnvVariables', () => {
+  let initialEnvironmentInfo: EnvironmentInfo;
+  let environmentInfo: EnvironmentInfo;
+
+  beforeAll(async () => {
+    initialEnvironmentInfo = await getEnvironmentInfo();
+  });
+
+  beforeEach(() => {
+    environmentInfo = initialEnvironmentInfo;
+  });
+
+  afterEach(() => {
+    jest.resetAllMocks();
+  });
+
+  it('returns a message if the Android SDK is not installed', async () => {
+    environmentInfo.SDKs['Android SDK'] = 'Not Found';
+    const diagnostics = await androidSDK.getDiagnostics(environmentInfo);
+
+    jest.mock('execa', () => ({
```

Comment:
I suggest mocking this at the top of the file, you can do:

```js
jest.mock('execa', () => ({
  default: jest.fn(),
});
```

And then, on each test case:

```js
execa.mockReturnValueOnce('my value');
```

I believe this would work.

Reply:
Thanks! That worked 😄 
I'm not an export on Jest mocks 🙄